### ISSUE TITLE
all: document that error.Unwrap was added in 1.13

### ISSUE THREAD

**Issue by rillig** [golang/go#36781]
### What version of Go are you using?

1.12 and 1.13

### What did you do?

To classify network errors, I used the `Unwrap` method:

~~~go
func classifyNetworkError(err error) string {
	cause := err
	for {
		type nested interface{ Unwrap() error }
		unwrap, ok := cause.(nested)
		if !ok {
			break
		}
		cause = unwrap.Unwrap()
	}
	return cause.Error()
}
~~~

The `Unwrap` method comes handy, I just wondered why there is no predefined interface like `NestedError` for that method.

I got strange differences between Go 1.12 and Go 1.13, which I couldn't explain. In Go 1.12 the errors were not unwrapped at all.

### What did you expect to see?

The `Unwrap` in all error types are clearly documented as "added in 1.13".

### What did you see instead?

Nothing.

### References

* #36767 
* commit 170b8b4b12be50eeccbcdadb8523fb4fc670ca72

**Comment by smasher164** [comment]
I looked through the Unwrap methods on exported types, listed here (https://golang.org/search?q=Unwrap), and notice that they all list "1.13" on godoc. What additional documentation do you have in mind here?

**Comment by rillig** [comment]
Oops, I must have missed that information. Or, which is more likely, I trusted my IDE to show me all relevant information, which it didn't.

When I looked at the implementation of the various `Unwrap` methods, I didn't see any version being mentioned. Therefore I thought it was missing. In addition, even after I found the commit that added the `Unwrap` methods, that commit did not add the entries to `api/go1.13.txt`, which I would have expected.

With all this additional information, the golang project is fine. I'll tell JetBrains to improve their IDE by marking the API level of each definition. Thanks for the help.
